infrastructure/lambda/template.yaml

```diff
@@ -659,7 +747,8 @@ Resources:
           POWERTOOLS_SERVICE_NAME: !Sub "${CriIdentifier}-session"
           SQS_AUDIT_EVENT_QUEUE_URL:
             Fn::ImportValue: !Sub ${TxmaStackName}-AuditEventQueueUrl
-          ENV_VAR_FEATURE_FLAG_KEY_ROTATION: !FindInMap [ KeyRotationMapping, !Ref CriIdentifier, !Ref Environment]
```

Comment:
We don't want to remove this one I don't think? Lime added it for the implementation of our own jwks endpoints piece - this is the OJ ticket https://govukverify.atlassian.net/browse/OJ-3143

Reply:
Good catch - reverted!